grakn-dist/src/dist.xml

```diff
@@ -28,82 +28,46 @@
     <fileSets>
         <fileSet>
             <outputDirectory>/</outputDirectory>
-            <directory>${main.basedir}</directory>
+            <directory>${basedir}/src</directory>
             <includes>
-                <include>CHANGELOG</include>
-                <include>LICENSE.txt</include>
-                <include>README.md</include>
+                <include>grakn</include>
+                <include>graql</include>
             </includes>
         </fileSet>
+        <!-- changelog license readme -->
         <fileSet>
-            <outputDirectory>src</outputDirectory>
-            <directory>${main.basedir}/</directory>
+            <outputDirectory>/</outputDirectory>
+            <directory>${main.basedir}</directory>
             <includes>
-                <include>pom.xml</include>
                 <include>LICENSE.txt</include>
-                <include>grakn-core/**</include>
-                <include>grakn-client/**</include>
-                <include>grakn-test-profiles/**</include>
-                <include>grakn-dashboard/**</include>
-                <include>grakn-dist/**</include>
-                <include>grakn-engine/**</include>
-                <include>grakn-kb/**</include>
-                <include>grakn-graql/**</include>
-                <include>grakn-graql-shell/**</include>
-                <include>grakn-migration/**</include>
-                <include>grakn-test/**</include>
-                <include>grakn-factory/**</include>
+                <include>README.md</include>
             </includes>
-            <excludes>
-                <exclude>**/target/</exclude>
-                <exclude>grakn-dashboard/node_modules/</exclude>
-                <exclude>grakn-dashboard/static/dashboard.js</exclude>
-            </excludes>
         </fileSet>
         <fileSet>
-            <outputDirectory>docs/api</outputDirectory>
```

Comment:
do we want to stop including docs with the dist? @fppt @aelred

Reply:
again I think, people would normally just go to the website or google for a tutorial. Let's wait for the others to chime in as well

Reply:
It will be useful for people running older versions of the software post-1.0.
Including docs mitigates against things like javadoc.io being unavailable, especially as we're not including the full source code with the dist anymore.

Also the REST API documentation is only available on our website and I pull it out of the dist.

Forgot to tag @pluraliseseverythings.

Reply:
Need more opinion

Current standing:
1. Don't include: Ganesh, Felix
2. Include: Thanh

Reply:
I personally have never looked at the java docs which come with the package so I say remove it. If you using the java API your IDE will likely show you the docs before you even think of checking the distribution.

Reply:
It's definitely useful if we only ever keep the last docs online. If we delete this we need to be able to access docs as grakn.ai/docs/1.0.0/ or grakn.ai/docs/0.15.0

Reply:
@pluraliseseverythings this is only removing the javadocs, which actually _are_ versioned (to my surprise :P )

see: https://static.javadoc.io/ai.grakn/grakn/0.12.0/overview-summary.html

Reply:
yes our javadocs are already online and versioned.

Reply:
Let's kill them then

Reply:
Got it